lib/crowbar/init/application.rb

```diff
@@ -392,9 +392,7 @@ class Application < Sinatra::Base
 
           if init[:body] # nil body means success
             logger.error(init[:body])
-            res[:errors] = {
-              data: init[:body]
-            }
+            res[:errors] = init[:body]
```

Comment:
What I do not understand is that above (line 383 e.g.) we are creating that errors[:data] structure. So what's the difference here?